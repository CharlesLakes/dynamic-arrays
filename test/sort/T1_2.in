10
60 88 78 12 21 12 61 16 81 5