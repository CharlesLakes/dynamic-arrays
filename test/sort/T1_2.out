9 12 15 16 20 25 41 49 52 53